app/assets/stylesheets/modules/sitter.scss

```diff
@@ -28,6 +28,9 @@
       }
     }
 
+    #sitter-info {
```

Comment:
Avoid using id selectors<br>Selector should have depth of applicability no greater than 2, but was 3